src/fights.ts

```diff
@@ -892,12 +892,44 @@ const freeFightSources = [
   ),
 
   new FreeFight(
-    () => have($item`[glitch season reward name]`) && !get("_glitchMonsterFights"),
-    () => {
-      restoreHp(myMaxhp());
-      retrieveItem($item`[glitch season reward name]`);
-      visitUrl("inv_eat.php?pwd&whichitem=10207");
-      runCombat();
+    () =>
+      have($item`[glitch season reward name]`) &&
+      !get("_glitchMonsterFights") &&
+      get("garbo_fightGlitch", false),
+    () =>
+      withMacro(
+        Macro.trySkill($skill`Curse of Marinara`)
+          .trySkill($skill`Conspiratorial Whispers`)
+          .trySkill($skill`Shadow Noodles`)
+          .externalIf(
+            get("glitchItemImplementationCount") >= 1000,
+            Macro.item([$item`gas can`, $item`gas can`])
+          )
```

Comment:
I'd change this to 
```
.externalIf(
  itemAmount($item`[glitch season reward name]`) * get("glitchItemImplementationCount") >= 1000,
  Macro.item([$item`gas can`, $item`gas can`])
)
``` at the least since both numbers are a factor for how strong the glitch monster is

Reply:
oh god dammit did I not push